.github/workflows/coveralls.yml

```diff
@@ -34,14 +36,13 @@ jobs:
           command: test
           args: --features "with_ctap1,with_nfc,std" --no-fail-fast
         env:
-          CARGO_INCREMENTAL: '0'
-          RUSTFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests'
-          RUSTDOCFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests'
-      - uses: actions-rs/grcov@v0.1.5
-        id: coverage
+          RUSTFLAGS: "-Zinstrument-coverage"
+          LLVM_PROFILE_FILE: "opensk-%p-%m.profraw"
+      - name: Run grcov
```

Comment:
You should be able to keep the `action-rs/grcov`. The parameters that are passed to the binary are stored in [.github/actions-rs/grcov.yml](https://github.com/google/OpenSK/blob/develop/.github/actions-rs/grcov.yml) file.

Otherwise please use long options. Short ones are meant for command line and may change between 2 releases. Long options are meant for scripts and should remain stable over time. And in this case, it might be worth looking if setting the `--vcs-branch` parameter is required to avoid the reports being mixed up by coveralls (might require some Github actions trickery to extract this piece of information though)

Reply:
Seems that `--vcs-branch` is not necessary. Long options are in. The `grcov` action doesn't seem to work for our use case.